package.json

```diff
@@ -23,11 +23,14 @@
     "build:readme": "mmarkdown",
     "build": "yarn build:cli && yarn build:cz",
     "clean": "rimraf dist binaries",
-    "eslint": "eslint lib/*.js",
-    "lint": "yarn eslint",
+    "lint:eslint": "eslint . --ext js,jsx,ts,tsx --max-warnings=0",
```

Comment:
A bit much on the extensions, can 100% remove `jsx|tsx` however would like to move project to TypeScript. So can stay for now and be removed later.